packages/ui/src/Autosuggest/types.ts

```diff
@@ -41,20 +41,21 @@ export type TSingleVariantProps = DiffAutosuggestProps<
   AutosuggestPropsSingleSection<ISuggestValue>
 >;
 
-export type AutosuggestProps<T = Record<string, unknown>> = Omit<
-  AutosuggestPropsBase<ISuggestValue>,
-  RewritedProps
-> &
+export type AutosuggestProps<
+  T = Record<string, unknown>,
+  TRewrittenProps extends string = RewrittenProps
+> = Omit<AutosuggestPropsBase<ISuggestValue>, TRewrittenProps> &
```

Comment:
мб лучше сделать дополняемым?
```suggestion
export type AutosuggestProps<
  T = Record<string, unknown>,
  ExtraRewrittenProps extends string = never
> = Omit<AutosuggestPropsBase<ISuggestValue>, RewrittenProps & ExtraRewrittenProps> &
```
мне кажетя, потребителю нежелательно иметь возможность изменять `RewrittenProps`